app/i18n/locales/en-US.json

```diff
@@ -168,8 +168,34 @@
   "wallet.restore.dialog.wallet.name.input.hint": "Enter wallet name",
   "wallet.restore.dialog.wallet.name.input.label": "Wallet name",
   "wallet.restore.dialog.walletPasswordLabel": "Wallet password",
-  "wallet.trezor.dialog.title.label": "Connect to Trezor Hardware Wallet",
-  "wallet.trezor.dialog.trezor.connect.button.label": "Connect",  
+  "wallet.trezor.dialog.title.label": "CONNECT TO TREZOR HARDWARE WALLET",
+  "wallet.trezor.dialog.trezor.step.about.label": "ABOUT",
+  "wallet.trezor.dialog.trezor.step.connect.label": "CONNECT",
+  "wallet.trezor.dialog.trezor.step.save.label": "SAVE",
+  "wallet.trezor.dialog.trezor.next.button.label": "Next",
+  "wallet.trezor.dialog.trezor.connect.button.label": "Connect",
+  "wallet.trezor.dialog.trezor.save.button.label": "Save",
+  "wallet.trezor.dialog.trezor.common.step.link.helpYoroiWithTrezor": "https://yoroi-wallet.com/",
+  "wallet.trezor.dialog.trezor.common.step.link.helpYoroiWithTrezor.text": "Click here to know more about how to use Yoroi with Trezor",  
+  "wallet.trezor.dialog.trezor.step.about.introText": "Trezor hardware wallet is a small USB device that allows you to access your wallet quickly, safely & easily. It is more secure because your private key never leaves the hardware wallet. It protects you from phishing, malware, and more.",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.header": "Prerequisite",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.1.part1": "Only Supports ",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.1.part2.link": "https://github.com/trezor/trezor-core/blob/master/ChangeLog",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.1.part2.link.text": "Trezor Model T with Version 2.0.8",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.1.part3": " or later",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.2": "Trezor device must be pre-initialized",
+  "wallet.trezor.dialog.trezor.step.about.prerequisite.3": "Computer must be connected to the Internet throught the process",
```

Comment:
The computer needs to be connected to the Internet through the entire process.

Reply:
how about using:
`through` -> `throughout`

The computer needs to be connected to the Internet throughout the entire process.

?

Reply:
i feel `entire` is not needed is we are using `throughout`

Reply:
`The computer needs to be connected to the Internet throughout the process`

*revised by KP

Reply:
Likewise:

`Trezor device must remain connected to the computer throughout the process`